### DISCUSSION TITLE
How to run llama in worker mode on remote?

### DISCUSSION THREAD

**Discussion by Egor3f** [mudler/LocalAI#3492] [Q&A]
Hello! Trying to launch this mode: https://localai.io/features/distribute/#without-p2p
Started remote worker with `local-ai worker llama-cpp-rpc -- -H 0.0.0.0 -p 25678`
Statred local instance with `LLAMACPP_GRPC_SERVERS="?.?.?.?:25678" local-ai run`
And local instance still tries to run model only locally. Worker mode don't work.

**Comment by pratikbin** [comment]
similar https://github.com/mudler/LocalAI/issues/3968
